contracts/domain/BosonTypes.sol

```diff
@@ -193,6 +193,9 @@ contract BosonTypes {
     struct Dispute {
         uint256 exchangeId;
         uint256 buyerPercent;
+        uint256 sellerPercent;
```

Comment:
Changing the `Dispute` struct is dangerous. There are implications for existing storage.

1) The new fields are not at the end of the struct, and so previously stored structs will be misinterpreted and/or clobbered when written to.
2) Even if they were at the end of the struct, it isn't always safe to add members to structs, depending upon how those structs are stored, i.e., in arrays vs mappings.

For these reasons, we would more likely want to associate these new fields with the dispute via a mapping and not change the struct and all the client code (including the graph) that relies on its current shape.

Reply:
Noted, will change it in the new PR.